src/cljs/main/broadfcui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -133,7 +134,8 @@
         (when editing?
           [comps/SidebarButton {:color :exception-state :margin :top
                                 :text "Cancel Editing" :icon :cancel
-                                :onClick #(stop-editing state)}])]))])
+                                :onClick #(do (react/call :load-new-method-template this (:prev-snapshot-id @state))
```

Comment:
1. If you stick this all in a `(fn [])`, you won't need the `do`.
2. Neater way to do this:
```cljs
(this :load-new-method-template (:prev-snapshot-id @state))
```